src/app/pages/storage2/components/pools-dashboard/widget-topology/widget-topology.component.html

```diff
@@ -22,7 +22,7 @@ <h3 mat-card-title>
   </mat-card-header>
 
   <mat-card-content fxFlex="100">
-    <div class="content-container" fxLayout="column" fxLayoutAlign="space-between">
+    <div class="content-container" fxLayout="column">
```

Comment:
Let's switch the `<div>` underneath with `*ngIf="!loading"` with `ng-container` like so
`<ng-container *ngIf="!loading" fxLayout="row" ...>`
Also bring back the `fxLayoutAlign="space-between"` here to fix the weird alignment of the `Data VDEVs` row.

Reply:
Added `<ng-container *ngIf="!loading">...</ng-container>`.
![image](https://user-images.githubusercontent.com/22980553/181904725-f0d2d538-68c7-490c-b419-9895f785052f.png)

A `fxLayoutAlign="space-between"` attribute was removed already.